.eslintrc.json

```diff
@@ -71,4 +71,4 @@
   "reportUnusedDisableDirectives": true,
   "noInlineConfig": true,
   "plugins": ["@typescript-eslint"]
-}
+}
```

Comment:
oops

Reply:
Do I need to change something here?

Reply:
maybe reverting would be nice?

Reply:
I had to fix a merge conflict. I'm not able to reverse that without recreating this conflict. At least I think so...

Reply:
Okay then, it's only space tho so yeah